exercises/G-numbers/exercise.js

```diff
@@ -1 +1,11 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+
+var numberOfMentors = 10, numberOfStudents = 100;
+
+// console.log(numberOfMentors + " & " + numberOfStudents);
+console.log("Number of students: " + numberOfStudents);
+console.log("Number of mentors: " + numberOfMentors);
+console.log("Total number of students and mentors: " + (parseInt(numberOfMentors) + parseInt(numberOfStudents)));
+
```

Comment:
You have solved the exercises in a very smart way, I reviewed your solution and I tested it out and compared it to my code you used fewer variables which is very nice, my only suggestion is that since "var" is an old method to store data, you would be better off using "let". In fact, I did the same, but the best way to use variables is by using "let" and "const".